ReactSkia/components/RSkComponentImage.h

```diff
@@ -8,7 +14,10 @@
 #include "ReactSkia/views/common/RSkImageCacheManager.h"
 
 #define DEFAULT_IMAGE_FILTER_QUALITY kLow_SkFilterQuality /*Skia's Defualt is kNone_SkFilterQuality*/
-
+#define DEFAULT_MAX_CACHE_EXPIRY_TIME 1800000 // 30mins in seconds 1800000
+#define NO_CACHE "no-cache"
```

Comment:
for following 3 macro , I would suggest to Prefix with RNS_ and suffix with _STR

For example RNS_NO_CACHE_STR